### PR TITLE
.NET Core 2+ Naming and Versioning; Asset rename

### PR DESCRIPTION
**Pull request by ghost**
New file name structure for the runtime and the CLI
per: https://github.com/dotnet/designs/issues/2
1. Branding changes
2. renaming of assets

Addresses:
https://github.com/dotnet/cli/issues/6712

@dotnet/dotnet-cli

### CODE REVIEW

**Review by dsplaisted — APPROVED** [review]
I didn't compare this to the spec to verify it matches, but this LGTM.

**Review by livarcocc — APPROVED** [review]

**Comment by livarcocc** [comment]
@MattGertz this is the renaming changes that Lee and Richard are driving.

**Comment by johnbeisner** [comment]
AskMode Approval: .NET Core 2+ Naming and Versioning; Asset rename
[Ask Mode]
• Link to GitHub issue for open source changes
https://github.com/dotnet/cli/issues/6712
• Link to GitHub dotnet:release/[version] pull request (see Git Prepping a PR from the Release Branch)
https://github.com/dotnet/cli/pull/6714
• Describe the issue and how it manifests to the customer.
New file name structure for the runtime and the CLI per: https://github.com/dotnet/designs/issues/2
• What is the impact on the customer?
Please refer to: https://github.com/dotnet/designs/issues/2
• Are there changes that cannot be made in GitHub? (security fixes, etc)
no
• Characterize potential risks introduced by the fix.
The rename of CLI assets have the potential to impact all consumers of those assets.
• Code reviewers:
dsplaisted, eerhardt, mellinoe, livarcocc
• What testing has been completed?
Testing thru builds on Jenkins thru the PR: https://github.com/dotnet/cli/pull/6714
Manual builds on Windows x64
Files:
dotnet-dev-2.0.0-preview2-006154-win-x64.exe
dotnet-dev-2.0.0-preview2-006154-win-x64.wixpdb
dotnet-dev-2.0.0-preview2-006154-win-x64.zip
dotnet-runtime-sdk-2.0.0-preview2-006154-win-x64.zip
dotnet-sdk-2.0.0-preview2-006154-win-x64.msi
dotnet-sdk-2.0.0-preview2-006154-win-x64.MsiTested.sentinel
dotnet-sdk-2.0.0-preview2-006154-win-x64.wixpdb
dotnet-sdk-2.0.0-preview2-006154-win-x64.zip
dotnet-sdk-debug-2.0.0-preview2-006154-win-x64.zip
dotnet-host-2.0.0-preview2-25324-03-win-x64.msi
dotnet-hostfxr-2.0.0-preview2-25324-03-win-x64.msi
dotnet-runtime-2.0.0-preview2-25324-03-win-x64.msi
• Which platforms are impacted?
Linux + Mac + Windows

@MattGertz

File: build/Branding.props [hunk 1]
```diff
@@ -13,16 +13,16 @@
                                    '$(Rid)' == 'fedora.24-x64' OR
                                    '$(Rid)' == 'opensuse.42.1-x64' ">$(Rid)</ProductMonikerRid>
     <ProductMonikerRid Condition=" '$(ProductMonikerRid)' == '' ">$(OSName)-$(Architecture)</ProductMonikerRid>
-    
+
     <ArtifactNameSdk>dotnet-sdk</ArtifactNameSdk>
     <ArtifactNameSdkDebug>dotnet-sdk-debug</ArtifactNameSdkDebug>
     <ArtifactNameCombinedHostHostFxrFrameworkSdk>dotnet-dev</ArtifactNameCombinedHostHostFxrFrameworkSdk>
-    <ArtifactNameCombinedFrameworkSdk>dotnet-sharedframework-sdk</ArtifactNameCombinedFrameworkSdk>
+    <ArtifactNameCombinedFrameworkSdk>dotnet-runtime-sdk</ArtifactNameCombinedFrameworkSdk>
```
Comment (eerhardt) [inline build/Branding.props]: I'm not sure I understand the point of this output.  Is it really used?  If yes, what for?  I think the name is super confusing now:  `dotnet-runtime-sdk`.  It only contains the `sdk`, `shared`, `additionaldeps`, and `store` folders, which isn't really usable by itself.  You need a host and hostfxr to run anything.

https://github.com/dotnet/cli/blob/6ecc1fc6c3f58bde065413cc869fb7f97d296143/build/package/Layout.targets#L15-L18

/cc @livarcocc 

Can we just delete this "artifact"?

Comment (livarcocc) [inline build/Branding.props]: I will be honest, I don't know what this is used for either. @piotrpMSFT any chance you know?

Comment (eerhardt) [inline build/Branding.props]: Opened https://github.com/dotnet/cli/issues/6718

**Comment by eerhardt** [comment]
@johnbeisner - the README links at https://github.com/dotnet/cli/tree/release/2.0.0#installers-and-binaries need to be updated for this change.

Can you make the update?

**Pull request merged**